experimental/Schemes/IdealSheaves.jl

```diff
@@ -428,9 +428,12 @@ end
 #  return W, spec_dict
 #end
 #
-
+function isone(I::IdealSheaf)
+  return all(x->isone(I(x)), affine_charts(scheme(X)))
```

Comment:
Did you want to write
  return all(x->isone(I(x)), affine_charts(scheme(I)))
(There is no X in this context and scheme(I) would make sense.)

Reply:
done